### PR TITLE
Condition is always 'false'

### PR DESCRIPTION
**Pull request by wangchenglong01**
/kind enhancement

#### What this PR does / why we need it:
Condition is always 'false'. Variable `commit` is assigned `gitCommit` whose value is fixed length.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@wangchenglong01 Thank you for raising pull request. how about add more information that condition is always `false`, like where `commit` is setup and length is fixed.

**Comment by wangchenglong01** [comment]
> @wangchenglong01 Thank you for raising pull request. how about add more information that condition is always `false`, like where `commit` is setup and length is fixed.

Ok, I will add more  details.

**Comment by wangchenglong01** [comment]
The more details has been added.

File: pkg/projectinfo/projectinfo.go [hunk 1]
```diff
@@ -32,17 +32,11 @@ var (
 
 func ShortAgentVersion() string {
 	commit := gitCommit
-	if len(gitCommit) > 7 {
-		commit = gitCommit[:7]
-	}
 	return GetAgentName() + "/" + gitVersion + "-" + commit
```
Comment (rambohe-ch) [inline pkg/projectinfo/projectinfo.go:35]: how about use `normalizeGitCommit()` to optimize this snippet, then as following:
```
return GetAgentName() + "/" + gitVersion + "-" + normalizeGitCommit(gitCommit)
```

**Comment by rambohe-ch** [comment]
> @rambohe-ch The more details has been added.

@wangchenglong01 Thanks for your reply. how about use `normalizeGitCommit` func to unify `gitCommit` usage. I have add a new comment, please take a look.

File: pkg/projectinfo/projectinfo.go (hunk 1 above)
Comment (wangchenglong01) [reply pkg/projectinfo/projectinfo.go]: The latest code has been commited. Optimize this snippet.

**Review by rambohe-ch — APPROVED** [review]

**Pull request closed without merging**